components/resources/commands.grdp

```diff
@@ -0,0 +1,472 @@
+    Focus Next Pane
+  </message>
+  <message name="IDS_IDC_FOCUS_PREVIOUS_PANE">
+    Focus Previous Pane
+  </message>
+  <message name="IDS_IDC_FOCUS_BOOKMARKS">
+    Focus Bookmarks
+  </message>
+  <message name="IDS_IDC_FOCUS_INACTIVE_POPUP_FOR_ACCESSIBILITY">
+    Focus Inactive Popup For Accessibility
+  </message>
+  <message name="IDS_IDC_FOCUS_WEB_CONTENTS_PANE">
+    Focus Web Contents Pane
+  </message>
+  <message name="IDS_IDC_SHOW_BOOKMARK_BAR">
+    Show Bookmark Bar
+  </message>
+  <message name="IDS_IDC_SHOW_HISTORY">
+    Show History
+  </message>
+  <message name="IDS_IDC_SHOW_BOOKMARK_MANAGER">
+    Show Bookmark Manager
+  </message>
+  <message name="IDS_IDC_IMPORT_SETTINGS">
+    Import Settings
+  </message>
+  <message name="IDS_IDC_EDIT_SEARCH_ENGINES">
+    Edit Search Engines
+  </message>
+  <message name="IDS_IDC_HELP_PAGE_VIA_KEYBOARD">
+    Show Help
+  </message>
+  <message name="IDS_IDC_SHOW_APP_MENU">
+    Show App Menu
+  </message>
+  <message name="IDS_IDC_DEV_TOOLS_INSPECT">
+    Dev Tools Inspect
+  </message>
+  <message name="IDS_IDC_UPGRADE_DIALOG">
+    Upgrade Brave
+  </message>
+  <message name="IDS_IDC_SHOW_AVATAR_MENU">
+    Show Avatar Menu
+  </message>
+  <message name="IDS_IDC_DEV_TOOLS_TOGGLE">
+    Dev Tools Toggle
+  </message>
+  <message name="IDS_IDC_SHOW_SETTINGS_RESET_BUBBLE">
+    Reset Settings
+  </message>
+  <message name="IDS_IDC_SHOW_SRT_BUBBLE">
+    Show SRT Bubble
+  </message>
+  <message name="IDS_IDC_TOGGLE_FULLSCREEN_TOOLBAR">
+    Toggle Fullscreen Toolbar
+  </message>
+  <message name="IDS_IDC_CUSTOMIZE_TOUCH_BAR">
+    Customize Touch Bar
+  </message>
+  <message name="IDS_IDC_SHOW_BETA_FORUM">
+    Show Beta Forum
+  </message>
+  <message name="IDS_IDC_TOGGLE_JAVASCRIPT_APPLE_EVENTS">
+    Toggle Javascript Apple Events
+  </message>
+  <message name="IDS_IDC_INSTALL_PWA">
+    Install PWA
+  </message>
+  <message name="IDS_IDC_SHOW_MANAGEMENT_PAGE">
+    Show Management Page
+  </message>
+  <message name="IDS_IDC_PASTE_AND_GO">
+    Paste And Go
+  </message>
+  <message name="IDS_IDC_SHOW_FULL_URLS">
+    Show Full Urls
+  </message>
+  <message name="IDS_IDC_CARET_BROWSING_TOGGLE">
+    Caret Browsing Toggle
+  </message>
+  <message name="IDS_IDC_PERFORMANCE">
+    Show Performance Settings
+  </message>
+  <message name="IDS_IDC_SHOW_BOOKMARK_SIDE_PANEL">
+    Show Bookmark Side Panel
+  </message>
+  <message name="IDS_IDC_TRANSLATE_ORIGINAL_LANGUAGE_BASE">
+    Translate Original Language Base
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_OPEN_ALL">
+    Bookmark Bar Open All
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_OPEN_ALL_NEW_WINDOW">
+    Bookmark Bar Open All New Window
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_OPEN_ALL_INCOGNITO">
+    Bookmark Bar Open All Incognito
+  </message>
+  <message name="IDS_IDC_BOOKMARK_BAR_OPEN_INCOGNITO">
+    Bookmark Bar Open Incognito
+  </message>
```

Comment:
Should these say `Private` instead of `Incognito`?

Reply:
They sure should!